python/docs/source/conf.py

```diff
@@ -121,9 +121,12 @@
 # built documents.
 #
 # The short X.Y version.
-version = 'master'
+import pyspark
+import re
+version = re.sub(r"(\.?)dev.*", "", pyspark.__version__)
+
 # The full version, including alpha/beta/rc tags.
-release = os.environ.get('RELEASE_VERSION', version)
```

Comment:
hm, probably we should revert these changes so we can tell it's built from development version etc. Maybe for other vendors too (?).